cyclonedx/__init__.py

```diff
@@ -22,4 +22,4 @@
 
 # !! version is managed by semantic_release
 # do not use typing here, or else `semantic_release` might have issues finding the variable
-__version__ = "8.3.0"  # noqa:Q000
+__version__ = "1.0.0"  # noqa:Q000
```

Comment:
please revert the changes from https://github.com/CycloneDX/cyclonedx-python-lib/pull/711/commits/ce3fe7f30bbfd74d00da69ca12c183d75d52e0ed 

background: you were working on a `main` branch in your fork. and had github workflows enabled (should be disabled in forks). this caused a triggering of a release process.

Reply:
this might do the trick
```sh
git checkout main
git pull
git reset --hard HEAD~1
git push -f 
```

Reply:
Thanks for letting me know. I turned off the workflow and reverted the commit. I will lock my `main`.

Sorry for lots of commits, I was trying to fix the DCO check.